src/files/Dockerfile

```diff
@@ -1,2 +1,4 @@
 FROM jkuri/abstruse
```

Comment:
we should change to `bleenco/abstruse`

Reply:
the new version will be `bleenco/abstruse_node_js`,  `bleenco/abstruse_java`, ... and so on.
for more info you can check https://hub.docker.com/u/bleenco/dashboard/